src/main/webapp/js/instructorFeedbackEdit/visibilityOptions.js

```diff
@@ -114,22 +129,10 @@ function showVisibilityCheckboxesIfCustomOptionSelected($containingForm) {
     }
 }
 
-var checkCheckboxCallback = function(index, checkbox) {
-    if (checkbox.checked === false) {
-        $(checkbox).trigger('change');
-    }
-    checkbox.checked = true;
-};
-
-var uncheckCheckboxCallback = function(index, checkbox) {
-    if (checkbox.checked === true) {
-        $(checkbox).trigger('change');
-    }
```

Comment:
Actually, the previous version (this deleted one) and the current one only differs from the absence of this if block. The `checkbox.checked = *;` part is identical. Does the removal of the if block affects something, or is it part of the fix?